### DISCUSSION TITLE
Welcome to RAVE Discussions!

### DISCUSSION THREAD

**Discussion by acids-ircam** [acids-ircam/RAVE#30] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share past experience with RAVE (training, usage, dataset)
  * ... or get help if needed :)

**Comment by bspeer1** [comment]
<img width="1280" alt="Screen Shot 2024-04-22 at 5 05 17 PM" src="https://github.com/acids-ircam/RAVE/assets/125216953/3b81b947-8e28-43ba-aa22-9fb520b9656e">
Should I be getting validation when this runs?
